theme/tsconfig.build.json

```diff
@@ -5,7 +5,7 @@
     "baseUrl": ".",
     "allowSyntheticDefaultImports": true,
     "lib": ["DOM", "ES2020"],
-    "module": "ESNext",
+    "module": "NodeNext",
```

Comment:
Build fails during deployment w/o this change
https://vercel.com/balancer/docs-v3/BDARbJqkUfEdCDtw1ffsMUcvH5qw?filter=errors